lab-brianna/lib/body-parser.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
```

Comment:
good start. ask yourself what this exported function is doing, and what it needs as inputs. what is it outputting? this will be handled by our express middleware, but knowing what is going on under the hood helps!